src/zarr/codecs/crc32c_.py

```diff
@@ -37,7 +37,7 @@ async def _decode_single(
         crc32_bytes = data[-4:]
         inner_bytes = data[:-4]
 
-        computed_checksum = np.uint32(crc32c(inner_bytes)).tobytes()
+        computed_checksum = np.uint32(crc32c(inner_bytes)).tobytes()  # type: ignore[arg-type]
```

Comment:
Locally, I see this without the type ignore

```
src/zarr/codecs/crc32c_.py:40: error: Argument 1 to "crc32c" has incompatible type "ndarray[Any, dtype[Any]]"; expected "Buffer"  [arg-type]
Found 1 error in 1 file (checked 66 source files)
```

I don't suppose anyone else gets that?

Reply:
I get these locally but not in our CI. I haven't bothered to figure out why that is.